UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaDataLayer.cs

```diff
@@ -190,9 +190,18 @@ public void MakeSlipperyAt(Vector3Int position, bool canDryUp = true)
 	}
 
 	/// <summary>
-	/// Release reagents at provided coordinates, making them react with world
+	/// Release reagents at provided coordinates, making them react with world + decide what it should look like
 	/// </summary>
-	public void ReagentReact(ReagentMix reagents, Vector3Int worldPosInt, Vector3Int localPosInt)
+	///
+	/*
```

Comment:
was  this a note for yourself while deving?

Reply:
looks like stuff from jamessimo

Reply:
well you took over. Clean it.